server/api/index.js

```diff
@@ -1,31 +1,26 @@
-const Shopify = require("shopify-api-node");
-const express = require("express");
-const axios = require("axios");
+const Shopify = require('shopify-api-node');
 
 const { SHOPIFY_SHOP_NAME, SHOPIFY_API_KEY, SHOPIFY_PASSWORD } = process.env;
-const router = express.Router();
 
 const shopify = new Shopify({
-  shopName: SHOPIFY_SHOP_NAME,
-  apiKey: SHOPIFY_API_KEY,
-  password: SHOPIFY_PASSWORD
+	shopName: SHOPIFY_SHOP_NAME,
+	apiKey: SHOPIFY_API_KEY,
+	password: SHOPIFY_PASSWORD,
 });
 
-shopify.on("callLimits", limits => console.log(limits, "test2"));
+const create = param => {
+	shopify.order
+		.create(param)
+		.then(data => console.log(data))
+		.catch(err => console.error(err));
+};
 
-router.post("/transaction", async (req, res) => {
```

Comment:
Let's still run the code inside this route. We want to calculate total every time a new order is received, which will be when the webhook POSTs to /transaction